docs/nodes/nodes-library/core-nodes/Compression/README.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /nodes/n8n-nodes-base.compression
+description: Learn how to use the Compression node in n8n
+---
+
+# Compression
+
+The Compression node is useful to compress and decompress files. You can either use the `gzip` or `zip` file format.
+
+
+## Node Reference
+
+- Operations
+	- Compress
+	- Decompress
+
+- Options
+    - ***Binary Property*** field: This field allows you to specify the name of the binary property
+    - ***Output Prefix*** field: This field allows you to specify a prefix for the name of the file gets generated
+    - ***Output Format*** field: This field allows you to select an output format. This field is displayed when 'Compress' gets selected from the ***Operation*** dropdown list. The following are the available formats:
+        - gzip
+        - zip
+    - ***File Name*** field: This field allows you to specify a file name. This field is displayed when 'zip' gets selected from the ***Output Format*** dropdown list.
+
+## Example Usage
+
+This workflow allows you to compress binary files to zip format. You can also find the [workflow](https://n8n.io/workflows/908) on n8n.io. This example usage workflow would use the following nodes.
+- [Start](../../core-nodes/Start/README.md)
+- [HTTP Request](../../core-nodes/HTTPRequest/README.md)
+- [Compression]()
+
+
+The final workflow should look like the following image.
+
+![A workflow with the Edit Image node](./workflow.png)
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+
+### 2. HTTP Request node (GET)
+
+This example workflow uses the HTTP Request node to fetch an image from a URL. You can also use the [Read Binary File](../../core-nodes/ReadBinaryFile/README.md) node to read a file from the path you specify.
+
+1. Enter the URL of the image in the ***URL*** field. For example, `https://docs.n8n.io/assets/img/final-workflow.f380b957.png`.
+2. Select 'File' from the ***Response Format*** dropdown list.
+3. Enter `workflow_image` in the ***Binary Property*** field.
+4. Click on ***Execute Node*** to run the node.
+
+In the screenshot below, you will notice that the HTTP Request node fetches the image from the URL. This image gets passed on as binary data to the next node in the workflow.
+
+![Using the HTTP Request node to fetch an image from a URL](./HTTPRequest_node.png)
```

Comment:
In all the screenshots for this workflow, we should show the Binary panel because the other panels don't display anything and might confuse the reader.